app/controllers/brexit_checker_controller.rb

```diff
@@ -2,6 +2,7 @@ class BrexitCheckerController < ApplicationController
   include AccountConcern
   include AccountBrexitCheckerConcern
```

Comment:
Could we merge the `AccountConcern` and the `AccountBrexitCheckerConcern`?  They were separate because not everything the checker needed was useful for the sessions controller, but that controller is gone now.

Reply:
done here: https://github.com/alphagov/finder-frontend/pull/2472/commits/1d011a03b21c778cb843c83841048e1af146c22f